dockstore-integration-testing/src/test/java/io/dockstore/client/cli/OrganizationIT.java

```diff
@@ -1314,6 +1312,25 @@ public void testBasicCollections() {
             fail("Was able to reject an approved collection");
         }
 
+        // version 8 and 9 belong to entryId 2
+        long versionId = 9L;
```

Comment:
Codacy found an issue: ['9L' is a magic number.](https://app.codacy.com/gh/dockstore/dockstore/pullRequest?prid=6286359)